### ISSUE TITLE
Clarification on the Emergence of Action Controllability During Training

### ISSUE THREAD

**Issue by ABaldrati** [OpenDriveLab/Vista#20]
Hi,

Thank you for your great work on this project. I have a few questions regarding the emergence of action controllability during the training process described in your paper:

1. **Emergence of Action Controllability**: Does the action controllability emerge abruptly at a certain point during the training, or does it gradually increase over time?

2. **First Observation of Controllability**: Can you recall at approximately which iteration the action controllability was first observed during the training process?

3. **Effect of LoRA Layers**: Have you observed the emergence of action controllability even without training the LoRA layers?

Thank you in advance for your responses!

**Comment by Little-Podi** [comment]
Hi Alberto, thanks for your nice questions.

> Does the action controllability emerge abruptly at a certain point during the training, or does it gradually increase over time?

I'm not sure, but I think it should gradually increase over time instead of a sudden emergence.

> Can you recall at approximately which iteration the action controllability was first observed during the training process?

Sorry, I cannot provide an accurate estimation. I only tested the model twice during the training of the released version. The first test occurred after 10K iterations since I added the action conditions. However, I failed to observe any notable effect at that time. Then I continued the training for an extremely long time, maybe around 100K steps (I remember it was close to the end of the training period that I reported). That’s when I found that the control was working. Afterwards, I further trained the model a little bit and obtained the final version.

> Have you observed the emergence of action controllability even without training the LoRA layers?

Yes, training without LoRA layers can acquire the action controllability as well. But LoRA tuning will lead to better visual quality.
